### ISSUE TITLE
FAST003 does not recognize parameter alias

### ISSUE THREAD

**Issue by spreeni** [astral-sh/ruff#13263]
Aliases for query parameters are not considered by FAST003, which leads to an incorrect error message.

The following valid code
```python
from typing import Annotated
from fastapi import FastAPI, Path

app = FastAPI()


@app.get("/documents/{documentId}")
def get_document(document_id: Annotated[str, Path(alias="documentId")]) -> str:
    return "Lorem Ipsum"
```
outputs: 

> FAST003 Parameter `documentId` appears in route path, but not in `get_document` signature
